backend/api/serializers/CreditTrade.py

```diff
@@ -180,6 +195,18 @@ def validate(self, data):
                             "been `{}`.".format(self.instance.status.status)
             })
 
+        # if the user is the respondent, they really shouldn't be modifying
+        # other fields. So check if those have changed
+        if self.instance.respondent == request.user.organization:
+            data['fair_market_value_per_credit'] = \
+            self.instance.fair_market_value_per_credit
+
+            data['number_of_credits'] = \
+            self.instance.number_of_credits
+
+            data['zero_reason_id'] = \
+            self.instance.zero_reason_id
+
```

Comment:
trade_effective_date, type, or compliance period?

might be easier just to load from the database and pass that as self.isntance, overwriting status?

Reply:
I was considering that. But I was concerned about the is_rescinded field. Since it's not really a status field, then that also needs to be added.

But thinking more about it, it's probably better than potentially causing more issues.